code/modules/karma/karma.dm

```diff
@@ -436,6 +423,14 @@ You've gained <b>[totalkarma]</b> total karma in your time here.<br>"}
 	else
 		to_chat(usr, "<span class='warning'>Your ckey ([dbckey]) was not found.</span>")
 
+/client/proc/queryErrorLogEntry(var/err = null)
```

Comment:
Instead of making two procs for this, can you instead just make the things between the bracket an argument you pass in?

Also, can you remove var/ from proc's parameter. They're already implied and not needed.